tests/unit/test_templatevars.py

```diff
@@ -107,20 +108,6 @@ def test_load_template_vars_from_string_and_file_2(self):
     unittest.main()
 
 
-def test_get_template_vars_installed_flow(monkeypatch):
-    """It works on an installed flow.
-
-    n.b. Does not attempt to test ``load_template_vars``
-    """
-    monkeypatch.setattr(
-        'cylc.flow.templatevars.load_template_vars',
-        lambda templatevars, templatevars_file: {'foo': 'bar'}
-    )
-    opts = SimpleNamespace(templatevars='', templatevars_file='')
-    result = get_template_vars(opts, '', names=('eg/runN', 'eg/runN'))
-    assert result == {'foo': 'bar'}
```

Comment:
As this seemed to be testing the logic of handling the `names` tuple at heart, I removed it. Not sure if it needs a replacement. @wxtim ?

Reply:
Looks unnecessary to me, given your changes: Good work. It's easy to write tests, much harder to get rid. 👍🏼